02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,29 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>My friggen page</title>
 </head>
 <body>
+  <h1>Dan Cumming</h1>
+  <section><em>1300 E Riverside Dr.</em></section>
+  <h2>Bio</h2>
+  <p>Raw denim unicorn jean shorts, leggings selvage gastropub drinking vinegar occupy tofu. Thundercats enamel pin venmo, tofu YOLO dreamcatcher tousled coloring book narwhal. Craft beer hot chicken green juice mlkshk, deep v kinfolk gastropub pug viral tbh butcher hoodie heirloom kale chips four loko. Waistcoat 90's deep v wayfarers stumptown flexitarian, swag la croix scenester poutine organic mustache. Activated charcoal hoodie microdosing poke selvage mustache iceland. Hella health goth typewriter celiac offal chambray. Mastlkjlher cleanse iPhone lo-fi fam, la croix cliche stumptown gochujang typewriter poutine 90's helvetica narwhal etsy readymade.
+  </p>
+  <h3>Hobbies</h3>
+  <ul>
+    <li>Motorcycles</li>
+    <li>Making this website</li>
+    <li>Wearing sunglasses at night</li>
+  </ul>
+  <h3>Stuff that gets me going on HBO</h3>
+  <ul>
+    <li><a href="http://www.imdb.com/title/tt2401256/">The Night Of</a></li>
```

Comment:
I would also put `li` tags here in their own line. Otherwise this might get confusing once we start adding more attributes and later JS logic.